src/util/extend.js

```diff
@@ -7,10 +7,6 @@
  * @return {Object} Merged object
  */
 export default function extend(dest, ...sources) {
-    sources.forEach(source => {
-        Object.keys(source).forEach(key => {
-            dest[key] = source[key];
-        });
-    });
-    return dest;
+    console.warn('util.extend is deprecated; use Object.assign');
+    return Object.assign(dest, ...sources);
```

Comment:
leave the existing implementation as it is.

Reply:
@thijstriemstra I think we should discuss solutions rather than stating one's opinion in an imperative way. Any reasons to keep the old implementation?

Reply:
Yes, we do not remove existing implementations before deprecating it for a couple releases, no matter what the code is. Don't you agree? Is there a huge problem with that? Or does it **have** to go? And I don't know what you mean with "stating one's opinion in an imperative way."

Reply:
Absolute adherence to a _certain way_ without regard to reasons for and against seems a bit bureaucratic =)

In programming, implementation is rarely significant — what matters is the interface. https://google.com/search?q=programming+interface+vs+implementation

Reply:
I do not agree, but I'll leave the decision up to @katspaugh.

Reply:
I have no strong preference. It’s gonna be removed anyway.

Reply:
> it would be less work to keep it at this point.

I have no problem doing the Right Thing and do more work.

>  agree it would be better not to touch it until removal.

Well, let's stick to that then. @mustafa0x please make the changes or @katspaugh please merge it. I am staying away from this pull request.

Reply:
Cool 👍 
Thanks, @thijstriemstra. I think everyone appreciates your ideal.
And thanks @mustafa0x for the PR.

Reply:
Thanks @katspaugh and @thijstriemstra!

Is there anything I need to do here?

Reply:
If you could restore the old implementation...

Reply:
Done...

Reply:
Thanks... 😁